src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs

```diff
@@ -513,7 +513,7 @@ public T[] ToArray()
         {
             // Short path if the bag is empty
             if (_headList == null)
-                return new T[0];
+				return Array.Empty<T>();
```

Comment:
Yes, there are already some of both kinds in the repo, for various things (including some that verify this for ToArray, e.g. there's a Queue.ToArray test that verifies not-sameness and which fails with this PR.)